src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/change_password.html.twig

```diff
@@ -0,0 +1,97 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% set passwordHelpMessage = 'Password should be at least %num% characters long.'|trans({ '%num%': 8 }, 'Admin.Advparameters.Help') %}
+{% set oldPasswordVars = change_password.old_password.vars|merge(old_password|default({})) %}
+{% set newPasswordFirstVars = change_password.new_password.children.first.vars|merge(new_password.first_options|default({})) %}
+{% set newPasswordSecondVars = change_password.new_password.children.second.vars|merge(new_password.second_options|default({})) %}
+<div class="form-group row">
+  <label class="form-control-label">
+    {{ 'Password'|trans({}, 'Admin.Global') }}
+  </label>
+  <div class="col-sm">
+    {# "Change password" button #}
+    {{ form_widget(change_password.change_password_button, {
+      label: 'Change password...'|trans({}, 'messages'),
+      attr: {
+        class: 'btn-outline-secondary js-change-password',
+      }
+    }) }}
+
+    <div class="card card-body js-change-password-block d-none">
+      {# Current password input #}
+      {{ ps.form_group_row(change_password.old_password, oldPasswordVars, {
+        label: 'Current password'|trans({}, 'messages'),
```

Comment:
Same here, could you show me a screenshot, please? I cannot find the string in the software...